gallery/templates/nav.html

```diff
@@ -61,4 +61,9 @@
             </div>
         </div>
     </nav>
+    <center>
```

Comment:
Also, the `<center>` tag is from HTML4, which some browsers render, but it isn't good practice to use, as it isn't officially a part of HTML5 (the current standard). This uses bootstrap, so you can probably just apply the `align-center` class to either the link or a wrapper `<div>` to center it.